magefiles/main.go

```diff
@@ -14,27 +15,87 @@ import (
 
 // BootstrapTools installs all tools needed tobuild and release Armada.
 // For the list of tools this will install, see tools.yaml in the root directory
+type ToolsList struct {
+	Tools []string
+}
+
```

Comment:
this type definition could be moved back inside `BootstrapTools` as it is not used anywhere else